var/spack/repos/builtin/packages/whizard/package.py

```diff
@@ -141,15 +146,19 @@ def configure_args(self):
             # todo: hoppet
             # todo: recola
             # todo: looptools
-            # todo: gosam
             # todo: pythia6
         ]
         args.extend(self.enable_or_disable("fastjet"))
+        args.extend(self.enable_or_disable("gosam"))
         args.extend(self.enable_or_disable("pythia8"))
         args.extend(self.enable_or_disable("lcio"))
         args.extend(self.enable_or_disable("lhapdf"))
         args.extend(self.enable_or_disable("openloops"))
 
+        if "+gosam" in spec:
+            args.append(f"--with-golem={spec['gosam-contrib'].prefix}")
+            args.append(f"--with-gosam={spec['py-gosam'].prefix}")
+            args.append(f"--with-qgraf={spec['qgraf'].prefix}")
```

Comment:
Are these actually used during configuration? Or is whizard also using environment variables instead now to find these?

Reply:
Whizard can use both `$GOSAM_DIR` and `--with-gosam`, not sure of the order of precedence. I'd have to doublecheck if they are needed, I put those in when debugging the missing `golem.la` whizard was expecting.

Reply:
Turns out that the spack sets up the dependencies correctly, these lines weren't needed. Good catch!